hpa/experiment.yaml

```diff
@@ -27,10 +27,9 @@ spec:
   metrics:
   - minimize: true
     name: p95-latency
```

Comment:
do you want to switch the optimized metric to p99 instead of p95? otherwise i am fine with it